### PR TITLE
Re-indent the Objective-C code to 2 spaces

### PR DESCRIPTION
**Pull request by matiaskorhonen**
The same as #2490 but with two space indents instead of four.

A minimal reformatting of the Objective-C code for readability using Xcode's re-indent feature (in Xcode: Editor → Structure → Re-indent).

This should make it a bit easier to work on the iOS/macOS native code…

### CODE REVIEW

**Comment by matiaskorhonen** [comment]
@TheAlmightyBob here's a two space version of the re-indentation PR

**Comment by TheAlmightyBob** [comment]
@matiaskorhonen Thanks! 
image
does seem a _little_ less-drastic than 
image
:)

File: apple/RNCWebView.m [hunk 1]
```diff
@@ -456,7 +456,7 @@ - (void)didMoveToWindow
   if (self.menuItems != nil) {
     UILongPressGestureRecognizer *longPress = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(startLongPress:)];
     longPress.delegate = self;
-
+
```
Comment (TheAlmightyBob) [inline apple/RNCWebView.m:459]: Not a big deal, but weird to be that an autoformatting tool would add unnecessary whitespace like this....

**Review by TheAlmightyBob — APPROVED** [review]
